### DISCUSSION TITLE
Issue updating from 2.3.100 to 2.3.110 - soup says running latest version

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#7509] [Unsupported Versions] [answered]
Hello all,

I'm running Security Onion 2.3.100 at the latest hotfix version on a distributed airgapped grid. I downloaded the 2.3.110 version today, checked the hash and then tried to launch it using soup -y -f <ISO>. I get the following output (see image):
![versionhotfix](https://user-images.githubusercontent.com/84025129/157738277-daa31c9e-728e-4a08-b624-27d4a00fe3f1.JPG)
 
The output of my /etc/soversion is 2.3.100 and /etc/sohotfix is "20220202 20220203 20220301".  I checked $UPDATE_DIR/VERSION which still shows the original 2.3.80 version I installed and the HOTFIX file is empty.

The portion of the soup file that seems to cause the error is this:

upgrade_check() {
  # Let's make sure we actually need to update.
  NEWVERSION=$(cat $UPDATE_DIR/VERSION)
  HOTFIXVERSION=$(cat $UPDATE_DIR/HOTFIX)
  [[ -f /etc/sohotfix ]] && CURRENTHOTFIX=$(cat /etc/sohotfix)
  if [ "$INSTALLEDVERSION" == "$NEWVERSION" ]; then
    echo "Checking to see if there are hotfixes needed"
    if [ "$HOTFIXVERSION" == "$CURRENTHOTFIX" ]; then
      echo "You are already running the latest version of Security Onion."
      exit 0
    else
      echo "We need to apply a hotfix"
      is_hotfix=true
    fi
  else
    is_hotfix=false
  fi 

}

Can someone please advise on what I might be doing wrong? I just applied the latest hotfix in the last couple of days.

TYIA!

**Comment by scarl3t0** [comment]
Update: Multiple reboots seemed to put the Manager back in it's place. Soup is running fine now!

**Comment by dougburks** [comment, marked as answer]
I suspect your system still had the previous ISO image mounted and when you rebooted it then prompted you for the new ISO image.
